### ISSUE TITLE
Audit of Adobe mentions on VA.gov

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#30850]
## Issue Description
We need to audit our benefit pages and R&S articles to find where we have mentions of sending users to the Adobe website to download the latest version of Acrobat Reader. We want to make sure that we use consistent and accessible language when doing this, mentioning both the target and the purpose. 

On [how to download and open a VA.gov PDF form](https://www.va.gov/resources/how-to-download-and-open-a-vagov-pdf-form/) the action link is "Get Acrobat Reader for free from Adobe." This is the language we should be using on all our pages. 

List pages with Adobe mentions here:

https://www.va.gov/records/download-va-letters/
https://www.va.gov/resources/how-to-download-and-open-a-vagov-pdf-form/

## Tasks
- [x] Audit site for mentions of going to the Adobe website to download Reader (Selina)
- [x] Update content so it matches R&S article (Selina)
- [x] Review and publish pages (Randi)

## Acceptance Criteria
- [x] All pages with Adobe mentions have been accounted for
- [x] All pages are updated with same language

**Comment by RLHecht** [comment]
Thanks @lunascoop I reviewed and published this page. Will close once I validate on live site. FYSA @bethpottsVADEPO

**Comment by RLHecht** [comment]
Page is live. Closing ticket.
